src/l500/l500-private.cpp

```diff
@@ -163,5 +164,35 @@ namespace librealsense
             auto res = _hwm.send( cmd );
             _record_action( *this );
         }
+
+        void freefall_option::enable( bool e )
+        {
+            if( e != is_enabled() )
+            {
+                if( !e  &&  is_true() )
+                    set( 0 );
+                _enabled = e;
+            }
+        }
+
+        hw_sync_option::hw_sync_option( hw_monitor& hwm, std::shared_ptr< freefall_option > freefall_opt )
+            : bool_option( false )
+            , _hwm( hwm )
+            , _freefall_opt( freefall_opt )
+        {
+        }
+
+        void hw_sync_option::set( float value )
+        {
+            bool_option::set( value );
+
+            // Disable the free-fall option if we're enabled!
```

Comment:
Maybe
// Free-fall and HW-Sync are mutually-exclusive. Toggle Free-fall to adjust